src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_Layout.cshtml

```diff
@@ -46,8 +46,8 @@
         </div>
     </footer>
 
-    <script src="~/lib/jquery/dist/jquery.js"></script>
-    <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.js"></script>
+    <script src="~/lib/jquery/dist/jquery.min.js"></script>
```

Comment:
Let's not mix concerns here. Generally we try to keep PR's to addressing one issue because it makes them easier to review.

Reply:
no worries, want me to revert?

Reply:
Yes please.

Reply:
done